sonar-db/src/main/java/org/sonar/db/permission/template/PermissionTemplateDao.java

```diff
@@ -116,31 +123,19 @@ public int countUsers(DbSession session, OldPermissionQuery query, Long template
     }
   }
 
+  @Deprecated
   public int countGroups(DbSession session, OldPermissionQuery query, long templateId) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the missing @deprecated Javadoc tag. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)